packages/terra-button-group/src/ButtonGroupButton.scss

```diff
@@ -1,14 +1,55 @@
 :local {
   .button-group-button {
+    background-color: var(--terra-button-group-button-background-color, #fff);
+    border-color: var(--terra-button-group-button-border-color, #c8cacb);
     border-radius: 0;
     border-style: solid;
     border-width: 1px;
+    box-shadow: var(--terra-button-group-button-box-shadow, none);
+    color: var(--terra-button-group-button-color, #434a4d);
+    font-weight: var(--terra-button-group-button-font-weight, 400);
     margin: 0;
     position: relative;
 
+    &.is-disabled,
+    &:disabled {
+      opacity: var(--terra-button-group-button-disabled-opacity, 0.3);
+    }
+
+    &.is-focused,
     &:focus {
```

Comment:
Remove the `&:focus` selector here. This is causing the focus styles to be applied on clicks, when they should only be applied on keyboard focus.

Reply:
Agree. https://github.com/cerner/terra-core/pull/1155/commits/553693e81282805d5c9d4db1168757c4fcbca411